proxy/invoke.go

```diff
@@ -22,9 +22,9 @@ func InvokeFunction(gateway string, name string, bytesIn *[]byte, contentType st
 
 	reader := bytes.NewReader(*bytesIn)
 
-	var timeout *time.Duration
+	var noTimeout *time.Duration
```

Comment:
It might be nice to add a comment here stating that this is created specifically to disable the function request timeout

Reply:
okay good idea, i'll change the variable name